### ISSUE TITLE
getActionButtons() fails for CustomTreeItems due to containers being undefined

### ISSUE THREAD

**Issue by mbehr1** [webdriverio-community/wdio-vscode-service#36]
I try to add a test for my extension that has a tree view. In the tree view I can access the tree and e.g. call getLabel().
But trying to get the action buttons fails with
`Cannot read properties of undefined (reading 'isExisting')`

imho accessed from here:
https://github.com/webdriverio-community/wdio-vscode-service/blob/54cb76ff746f01948dacca9cef8cd9209e827b42/src/pageobjects/sidebar/ViewItem.ts#L135

Used versions:
mac (arm)
wdio-vscode-service 4.0.5
VSCode v1.70.2 (vscode-darwin-arm64-1.70.2 stable)  and Chromedriver v100.0.4896.60 

Any help appreciated!
(I tried the same/similar tests with vscode-extension-tester (just noticed that I tried it there with vscode 1.67.0) and can access the tree-view and trigger the action buttons without issues. But I'd prefer to use wdio-vscode-service as it seems to be magnitudes faster :-)

**Comment by mbehr1** [comment]
I tested with vscode 1.67.0 and it doesn't work either.

**Comment by mbehr1** [comment]
If I replace the function getActionButtons() with something like
```ts
const getActionButtons = async (itemElem: TreeItem) => {
  await itemElem.elem.moveTo();
  const container = await itemElem.elem.$(itemElem.locatorMap.TreeItem.actions as string);
  
  if (!await container.isExisting()) {
      return [];
  }
  
  const actions: ViewItemAction[] = [];
  const items = await container.$$(itemElem.locatorMap.TreeItem.actionLabel as string);
  
  for (const item of items) {
      const label = await item.getAttribute(itemElem.locatorMap.TreeItem.actionTitle as string);
      actions.push(new ViewItemAction(itemElem.locatorMap, label, itemElem));
  }
  return actions;
 };
```
I can get the ViewItemActions for it.
(They still don't offer a "click()" functionality but at least the label.)

**Comment by mbehr1** [comment]
`twistie` is not available as well. So it might be better to add the locators from TreeItem to CustomTreeItems (merge during constructor/decorator or use an array with [TreeItemLocator, CustomTreeItemLocator]?)

**Comment by mbehr1** [comment]
@christian-bromann any idea on this issue? Any hint on how to fix or preference/direction for a PR?

**Comment by christian-bromann** [comment]
Unfortunately I haven't really used that PO yet so I have no recommendation here. I am happy to review/merge any PR that will make this section actually useable.

**Comment by mbehr1** [comment]
Ok. I'll try a PR with:
- adding a tree view to the test extension
- adding a test for it
- adding a fix

**Comment by christian-bromann** [comment]
Sounds awesome, thanks! Let me know if you have any questions or if I can help somehow.

**Comment by christian-bromann** [comment]
Fixed in #39
